src/utils/fluxNode.ts

```diff
@@ -109,7 +109,12 @@ export function addUserNodeLinkedToASystemNode(
 
 export function modifyFluxNode(
   existingNodes: Node<FluxNodeData>[],
-  { asHuman, id, text }: { asHuman: boolean; id: string; text: string }
+  {
+    asHuman,
+    id,
+    text,
+    label,
+  }: { asHuman: boolean; id: string; text: string; label?: string }
```

Comment:
as mentioned earlier would prefer if modifying the label was a separate function